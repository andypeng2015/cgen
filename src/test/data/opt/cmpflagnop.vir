module foo

export function w $foo(w %x) {
@start:
  %c1 = lt.w %x, 5_w
  %f1 = flag.w %c1
  %c2 = ge.w %f1, 1_w
  %f2 = flag.w %c2
  ret %f2
}
