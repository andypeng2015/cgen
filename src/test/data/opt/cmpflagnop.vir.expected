module foo

export function w $foo(w %x) {
@0:
  %1 = lt.w %x, 0x5_w ; @6
  %0 = flag.w %1 ; @5
  ret %0
}
